content/docs/alerting/notifications.md

```diff
@@ -24,7 +24,9 @@ Note that some fields are evaluated as text, and others as HTML which will affec
 | ------------- | ------------- | -------- |
 | Receiver | string | Defines the receiver's name that the notification will be sent to (slack, email etc.). |
 | Status | string | Defined as firing if at least one alert is firing, otherwise resolved. |
-| Alerts | [Alert](#alert) | List of alert objects ([see below](#alert)). |
+| Alerts | [Alert](#alert) | List of all alert objects in this group ([see below](#alert)). |
+| Alerts.Firing | [Alert](#alert) | List of currently firing alert objects in this group ([see below](#alert)). |
```

Comment:
This isn't a field of data, it's actually a function of the `Alerts` type. I'd mention it below the table, rather than try to squeeze it in here.